static/js/play.js

```diff
@@ -35,6 +36,11 @@ let app = new Vue({
 
         home: function (event) {
             window.location.replace("/");
+        },
+
+        startNow: function (event) {
+            app.$data.started = true;
```

Comment:
I think you need to do 2 more things here:
1. Stop the countdown timer so startTime doesn't get reset (see [here](https://developer.mozilla.org/en-US/docs/Web/API/clearInterval)
2. Set the startTime here. I'm not sure why it's not inside the vue instance, but no need to worry about that now

Reply:
nvm about 2, it seems that we start when we load. However we don't need to set it below either even for quick start